x-pack/plugins/siem/public/resolver/view/process_event_dot.tsx

```diff
@@ -251,8 +256,33 @@ const ProcessEventDotComponents = React.memo(
      * position to accomodate for that. This aligns the logical center of the process node
      * with the correct position on the map.
      */
-    const processNodeViewXOffset = -0.172413 * logicalProcessNodeViewWidth * magFactorX;
-    const processNodeViewYOffset = -0.73684 * logicalProcessNodeViewHeight * magFactorX;
+
+    const isZoomedOut = magFactorX < 0.8;
+    const isShowingDescriptionText = magFactorX >= 0.55;
+
+    const nodeXOffsetValue = isZoomedOut
+      ? -0.143413 + (-magFactorX * (1 - magFactorX)) / 10
```

Comment:
can you explain these magic numbers, and this expression.

Reply:
I just modified the initial numbers set on 254 and 255 to better align on the graph. The function is just another magFactor dependent scale to help keep the position of the node centered as you scaled down and the event drop downs are removed from view.